site/source/tutorials/005_form_widgets/index.md

```diff
@@ -58,107 +77,128 @@ Let's use those classes and a few virtual DOM nodes, to add the visual elements
+
+The last change that needs to be made in the `WorkerForm` is to update the `_onSubmit` method to delegate to the `onFormSave` property when it is called.
 
-The last change that needs to be made in the `WorkerForm` is to update the `_onSubmit` method to delegate to the `onSubmit` property when it is called. Replace the `_onSubmit` method with:
+{% instruction 'Replace the `_onSubmit` method with.' %}
 
-{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:26-31 %}
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:26-29 %}
 
-## Integrating the WorkerForm into the App
+The form is now ready to be integrated into the application, we will do that in the next step.
 
-Now that the `WorkerForm` widget is complete, we will update the `App` class to use it. First, we need to address how to store the user-completed form data. Recall that the `WorkerForm` will accept an `onChange` property that will allow the `App` class to be informed when a field value changes. However, the `App` class does not currently have a place to store those changes. We could add private fields in the class to store those values, but that would be difficult to maintain over time as the application gets more complicated and more data needs to be stored. Instead, we are going to take advantage of another mixin that comes with Dojo 2 that is designed to handle state for us - the `StatefulMixin`.
+{% section %}
+
+## Using forms
+
+{% task 'Integrate the form into the application' %}
+
+Now that the `WorkerForm` widget is complete, we will update the `App` class to use it. First, we need to address how to store the user-completed form data. Recall that the `WorkerForm` will accept an `onFormInput` property that will allow the `App` class to be informed when a field value changes. However, the `App` class does not currently have a place to store those changes. We could add private fields in the class to store those values, but that would be difficult to maintain as the application gets more complicated and more data needs to be stored. Instead, we are going to take advantage of another mixin that comes with Dojo 2 that is designed to handle state for us - the `StatefulMixin`.
```

Comment:
"gets more complicated and more data needs to be stored" complicated feels like the wrong term, more advanced, more feature-rich, more ?